frontend/app/src/types/collection.ts

```diff
@@ -1,4 +1,12 @@
-import { type BigNumber } from '@rotki/common';
+import { type BigNumber, NumericString } from '@rotki/common';
+import { z } from 'zod';
+
+export const CollectionGenericType = z.object({
```

Comment:
nitpick: since generic usually means something else in the language context maybe CollectionCommonFields would be less confusing what do you think?